src/linker/Linker/SubstitutionInfo.cs

```diff
@@ -0,0 +1,45 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Mono.Cecil;
+
+namespace Mono.Linker
+{
+	public class SubstitutionInfo
+	{
+		public Dictionary<MethodDefinition, MethodAction> MethodActions;
+		public Dictionary<MethodDefinition, object> MethodStubValues;
+		public Dictionary<FieldDefinition, object> FieldValues;
+		public HashSet<FieldDefinition> FieldInit;
```

Comment:
Does it need to be public when you already have several Set* methods?

Reply:
Thanks, these should have been properties (they need to be public)